csharp/Platform.IO/FileHelpers.cs

```diff
@@ -28,9 +76,49 @@ public static T ReadFirstOrDefault<T>(string path)
             return fileStream?.ReadOrDefault<T>() ?? default;
         }
 
+        /// <summary>
+        /// <para>Returns the <see cref="FileStream"/> with opened file from the file at <paramref name="path"/> if file exists, not empty and its size is a multiple of the <typeparamref name="TStruct"/> structure size, otherwise <see langword="null"/>.</para>
```

Comment:
`the <see cref="FileStream"/> with opened file` ⇾ `the <see cref="FileStream"/> opened for reading`

Reply:
`if file exists` => `if the file exists`